websockets.md

```diff
@@ -267,7 +267,7 @@ If the client specifies an invalid query parameter or JSON field for the recogni
 
 After it sends the initial `start` message, the client can begin sending audio data to the service. The client does not need to wait for the service to respond to the `start` message with the `listening` message. The service returns the results of the transcription asynchronously in the same format as it returns results for the HTTP API.
 
-The client must send the audio as binary data. The client can stream a maximum of 100 MB of audio data with a single utterance (per `send` request). The client can send multiple utterances over a single WebSocket connection.
+The client must send the audio as binary data. The client can stream a maximum of 100 MB of audio data with a single utterance (per `send` request). It must send at least 100 bytes of audio for any request. The client can send multiple utterances over a single WebSocket connection.
```

Comment:
Probably a ding from acrolinx on the `It` backward reference. Possible: "Each request must send at least 100 bytes."?

Reply:
But the backward ref is to "The client", which is the subject of both of the previous sentences.  if acrolinx doesn't like this, oh well.... :-)